docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "2"
+services:
+  web: 
+    build: .
+    ports: 
+      - 80:3000
+    command: bash -c 'while !</dev/tcp/db/5432; do sleep 1; done; npm run start:dev'
+    depends_on:
+      - db
+    environment:
+      - DATABASE_URL=postgres://forum@db:5432/forum
+      - PORT=3000   
+      - FORUM_STORAGE_ADAPTER=S3
+    volumes:
+      - ./app:/code/app
+      - ./stylesheets:/code/stylesheets
+  db:
+    build:
+      context: .
+      dockerfile: ./Dockerfile.postgres
+  # Testing
+  e2e:
```

Comment:
is `e2e` a reserved word that docker or travis uses to pin these statements to particular hooks? is it a convention?

if neither, can we call this `test` to match the `test_db` below?

Reply:
Good call. f383020